config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
 
   root 'welcome#index'
+  resource :session
+
+  get '/jokes/random_joke' => "jokes#random_joke"
+  get '/jokes' => 'jokes#index'
+  post '/jokes/current_joke' => "jokes#current_joke"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.